### ISSUE TITLE
Squished positive/neg split graph

### ISSUE THREAD

**Issue by domyen** [StateOfJS/StateOfJS-2020#12]
The positive/neg split labels overlap each other. It appears to be because the height of the bar defines the height of the row.  

I realize that this is probably intentional, just figured I'd call it out. Perhaps the minimum bar height could be the height of the label?

![image](https://user-images.githubusercontent.com/263385/104258097-e0299000-544c-11eb-9e33-f1d4e5c0311a.png)

**Comment by SachaG** [comment]
Yeah I'm not sure how to avoid that without messing with the bar height, which would make them not accurate… Maybe in the future we need to rethink that chart or space out the bars more.
